src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = [...moviesFromServer].filter(
```

Comment:
For what spreading here?
Filter  is immutable method and return new filtered array.

Reply:
Yeah, sure, thanks!